tools/get-teams-devops.sh

```diff
@@ -0,0 +1,116 @@
+#!/bin/bash
+# This script verifies the successfull deployment of the resources needed for the DevOps OpenHack.
+# The error log file is where will be logged the informations regarding the failed deployments. If not provided, it defaults to error.log. 
+
+usage() { echo "Usage: getteams.sh -f <errorlog_file> -u <service principal username> -f < service principal password> -x < service principal tenant>" 1>&2; exit 1; }
+
+while getopts ":f:u:x:t:" arg; do
+    case "${arg}" in
+        f)
+            ERROR_FILE=${OPTARG}
+        ;;
+        u)
+            USERNAME=${OPTARG}
+        ;;
+        x)
+            PASSWORD=${OPTARG}
+        ;;
+        t)
+            TENANT=${OPTARG}
+        ;;
+    esac
+done
+
+KEY_DIR=~/devopsohkeys
+mkdir -p ${KEY_DIR}
+ID_RSA_PRIVATE=${KEY_DIR}/devops_openhack_key
+ID_RSA_PUBLIC=${KEY_DIR}/devops_openhack_key.pub
+ssh-keygen -t rsa -C "DevOps OpenHack" -f ${ID_RSA_PRIVATE} -P ""
+if [ $? -ne 0 ]; then
+    echo "[ERROR] Creating the ssh keypair for the proctorVM failed "
+fi
+
+echo "Keys have been generated"
+read ID_RSA_PUBLIC
+
+if [[ -z "$ERROR_FILE" ]]; then
+    ERROR_FILE="./errors.log"
+fi
+
+DEPLOYMENTSTATUS=0
+
+touch $ERROR_FILE
+
+az login --service-principal -u $USERNAME -p $PASSWORD --tenant $TENANT
+
+ipaddress=$(az vm list-ip-addresses --resource-group=ProctorVMG --name=proctorVM --query "[].virtualMachine.network.publicIpAddresses[].ipAddress" -otsv)
+echo IPADDRESS:$ipaddress
+
+if [[ $ipaddress =~ ^[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}$ ]]; then
+    teamAAD=$(echo "aprao"  | sed -e 's/.*?*@\(.*\)ops.onmicrosoft.com/\1/')
+    echo TEAM:$teamAAD
+    if [[ ! -d "$teamAAD" ]]; then
+    mkdir -p $teamAAD
+    fi
```

Comment:
@dcaro I am still not sure what teamAAD should be from yesterdays discussion. Just a simple name and remove the other stuff?